src/installer-skin.lib.php

```diff
@@ -51,10 +51,12 @@ class SucuriScanPluginInstallerSkin extends Plugin_Installer_Skin
             /**
              * Reports the progress of the plugin installation.
              *
-             * @param  string $string Message to send to the buffer.
+             * @param string $string  Message to display in administration message.
+             * @param mixed  ...$args Optional text replacements.
+             *
              * @return void
              */
-            public function feedback($string = '')
+            public function feedback($string = '', ...$args)
```

Comment:
You're right @gvre, thanks! What do you think about having two versions of the `SucuriScanPluginInstallerSkin` class? We can have a default version with support for PHP 5.6+, and a legacy version with support for older versions of PHP.

This won't be the most elegant solution, but it'll keep the same experience for legacy and recent PHP versions.

Reply:
At least until we drop support for PHP prior 5.6.

Reply:
Having two versions of the `SucuriScanPluginInstallerSkin` class seems to be a good, short-term solution.
It seems [that breaking change introduced in WP 5.3](https://github.com/WordPress/WordPress/commit/340b7b53c88e35f103fb20f3b5c0cc87febaf8f8#diff-20e8256fd10ad59167b3ab405d250a73738c316daf99c3a40da497c05c88d9bfR55), so I think it makes sense to check the installed WP version, instead of the PHP one.

Reply:
Great finding, thanks! I will update the PR 👍🏾

Reply:
@gvre I addressed this in https://github.com/Sucuri/sucuri-wordpress-plugin/pull/117/commits/9d4e4c80d92621f9bde57061808f6bef61ea1fa8 and https://github.com/Sucuri/sucuri-wordpress-plugin/pull/117/commits/444c4bcf1689dd989557fc1bd024344f2ef054e2 :)

Reply:
```public function feedback($string = '', ...$args)```
@imgerson This will probably trigger a deprecation warning in PHP 8 (see #110). If it does, you can remove the default value from the first parameter, in order to make it required.

Reply:
@gvre I didn't find the warning in the logs, but noticed [here](https://github.com/WordPress/WordPress/commit/340b7b53c88e35f103fb20f3b5c0cc87febaf8f8#diff-20e8256fd10ad59167b3ab405d250a73738c316daf99c3a40da497c05c88d9bfR55) that the `$string` param was never optional, so I pushed https://github.com/Sucuri/sucuri-wordpress-plugin/pull/117/commits/694731fc4e79cb43d9ff9e17cae3af4ce0a4407b updating both classes.